drivers/memc/CMakeLists.txt

```
@@ -28,3 +28,6 @@ zephyr_library_sources_ifdef(CONFIG_MEMC_SMARTBOND memc_smartbond_nor_psram.c)
 
 zephyr_library_sources_ifdef(CONFIG_MEMC_MSPI_APS6404L memc_mspi_aps6404l.c)
 zephyr_library_include_directories_ifdef(CONFIG_MEMC_MSPI_APS6404L ${ZEPHYR_BASE}/drivers/mspi)
+
+zephyr_library_sources_ifdef(CONFIG_MEMC_MSPI_APS256N memc_mspi_aps256n.c)
+zephyr_library_include_directories_ifdef(CONFIG_MEMC_MSPI_APS256N ${ZEPHYR_BASE}/drivers/mspi)
```

I was not very happy with this.
Could be aps6404 and aps256n all in one
```suggestion
zephyr_library_include_directories_ifdef(CONFIG_MEMC_MSPI ${ZEPHYR_BASE}/drivers/mspi)
```